homeassistant/components/meteo_lt/coordinator.py

```diff
@@ -39,23 +40,38 @@ def __init__(
             config_entry=config_entry,
         )
 
+    def _create_update_failed(self, message: str) -> UpdateFailed:
+        """Log unavailability once and create UpdateFailed exception."""
+        if not self._unavailable_logged:
+            _LOGGER.info(message)
+            self._unavailable_logged = True
+        return UpdateFailed(message)
```

Comment:
The logging only once is already happening by the coordinator, so no need to add this extra logic